src/tiled/mapobjectmodel.h

```diff
@@ -101,6 +110,7 @@ private slots:
     void layerChanged(Layer *layer);
     void layerAboutToBeRemoved(GroupLayer *groupLayer, int index);
     void tileTypeChanged(Tile *tile);
+    void emitObjectsDataChanged(const QList<MapObject *> &objects);
```

Comment:
I think this can be a private function, and I'd prefer the name to be simply `emitDataChanged`, since the parameter already makes clear that it is for objects.